python/lsst/ts/standardscripts/auxtel/enable_atcs.py

```diff
@@ -102,20 +103,13 @@ def get_schema(cls):
                       - type: string
                       - type: "null"
                     default: null
+                ignore:
+                    description: >-
+                        CSCs from the group to ignore. Name must match those in
+                        self.group.components, e.g.; {self.group.components}.
```

Comment:
Please consider "e.g. Hexapod:1" instead of "e.g.; {self.group.components}." and look for the same text elsewhere. (Even if this does expand to a useful example, I would argue that it is more useful to be able to read the example when reading the source code).